kythe/cxx/extractor/bazel_artifact_selector.h

```diff
@@ -168,13 +174,19 @@ class AspectArtifactSelector final : public BazelArtifactSelector {
   absl::Status DeserializeFrom(const google::protobuf::Any& state) final;
 
  private:
+  struct FileSet {
+    std::vector<const BazelArtifactFile*> files;
```

Comment:
The `files` hash_map owns these objects, right?

Reply:
Yeah, that's right. I'll add a comment to that effect in a separate PR.